middlewares/checkAuth.js

```diff
@@ -1,15 +1,36 @@
 import passport from "passport";
 import constants from "../helpers/constants";
+import models from "../models";
```

Comment:
Nice implementation for checking roles and permissions.
Can you refractor the code and create 2 separate middlewares one for checking authentication and another one for checking roles and permission?

Reply:
Hi @espoirMur, 

Thanks for the suggestions but if you look closely you will find that the codes in the file are doing the same job. Protecting the routes.
Thanks